sky/engine/core/css/CSSFontSelector.cpp

```diff
@@ -43,14 +43,13 @@ namespace blink {
 
 CSSFontSelector::CSSFontSelector(Document* document)
     : m_document(document)
-    , m_genericFontFamilySettings(document->frame()->settings()->genericFontFamilySettings())
 {
+    if (m_document)
+        m_genericFontFamilySettings = document->frame()->settings()->genericFontFamilySettings();
```

Comment:
what are generic font family settings and what's our story for getting some without documents?